HealthcareSystem/GraphicalEditorServer/Controllers/EquipmentController.cs

```diff
@@ -42,8 +42,17 @@ public ActionResult AddEquipment([FromBody] Equipment equipment)
         [HttpGet("byRoomNumber/{roomNumber}")]
         public IActionResult GetEquipmentByRoomNumber(int roomNumber)
         {
-            List<Equipment> equipmentsInRoom = _equipmentService.GetEquipmentByRoomNumber(roomNumber);
-            return Ok(equipmentsInRoom);
+            try
+            {
+                List<Equipment> equipmentsInRoom = _equipmentService.GetEquipmentByRoomNumber(roomNumber);
+                if (equipmentsInRoom == null) {
+                    return NotFound("NotFound");
+                }
```

Comment:
Again, why null? Empty list makes more sense and it'd be easier to work with.

Reply:
I have changed that part